.gitignore

```diff
@@ -27,3 +27,6 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+# Ignore environment files containing secrets for local execution of terraform
+.env
```

Comment:
@pratishthavermattn  This should not have been approved. Each file should have a new line character.